code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -31,9 +31,8 @@
 
 		//Machine-only emotes
 		if("ping", "beep", "buzz", "yes", "ye", "no", "rcough", "rsneeze")
-
-			if(!isSynthetic())
-				to_chat(src, "<span class='warning'>You are not a synthetic.</span>")
+			if(!isSynthetic() && src.can_speak(/datum/language/machine)) // whaaat the fuck is this
```

Comment:
Not sure if this would work, I think the check here should logically be

`if the user is NOT synthetic and CANNOT speak EAL...`

Should probably be `if(!isSynthetic() && !can_speak(/datum/language/machine))`
`

Reply:
fucked up that you didnt do a code suggestion so i could just have you coauthor it but i can work w/ that